### DISCUSSION TITLE
Hyper-V

### DISCUSSION THREAD

**Discussion by wardellcastles** [Genymobile/feedback#73] [Genymotion Desktop]
If the development team wants a Beta tester for Genymotion using Hyper-V, let me know!  email me at wardellcastles@red7en.com

**Comment by agavignet** [comment]
Hi @wardellcastles,

Thanks for your help. Did you try QEMU hypervisor on Windows already ([doc](https://docs.genymotion.com/desktop/Get_started/014_Basic_steps/#select-a-hypervisor))? This hypervisor relies on Hyper-V but we noticed a slowness once OpenGApps is installed, do you?

Adrien

**Comment by wardellcastles** [comment]
Adrien,

I have started using the QEMU Hypervisor.  You are right.  Once GAPPS is installed the UI comes to a crawl.  Before then though the UI was very and pleasingly responsive.   ADB installs were blazingly quick.  The DIVA app was very responsive.    I hope you can resolve the GAPPS issue because other than that, I much prefer that Hypervisor over Oracle Virtual Box.

I will give you more feedback in the coming days.

Wardell

**Comment by wardellcastles** [comment]
Adrien,

It is my experience using the QEMU hypervisor that the UI experience is very unpredictable.

One time the UI experience is very satisfactory.  However, I can start the device again and the UI is completely unresponsive and the device crashes.

I am unable to determine at this time what causes the UI experience to be unpredictable.

**Reply by jmaneyrol69** [reply]
CC @agavignet

**Comment by agavignet** [comment]
> One time the UI experience is very satisfactory. However, I can start the device again and the UI is completely unresponsive and the device crashes.

Thanks for reporting your experience :) You mean even without OpenGApps installed or with OpenGApps?

**Comment by wardellcastles** [comment]
With Open GAPPS.

I will create another device and not install Open GAPPS and see how the UI acts.

**Comment by wardellcastles** [comment]
Here is a different issue.

I created a device with Android 12 - API 31.  While the device starts and appears to be performing normally the Status in Genymotion stays at "Booting".

Attempts to access the device using adb show that the device is offline.
List of devices attached
127.0.0.1:6555  offline

One thing I have noted in the other devices I have created that do work is that the IP address is 127.0.0.1:65555   Is this correct?

OpenGapps is not installed.  Indeed that option is greyed out on the option bar.
